examples/code-splitting-native-import-context/template.md

```diff
@@ -0,0 +1,51 @@
+# example.js
+
+This example illustrates how to leverage the `import()` syntax to create ContextModules which are separated into separate chunks for each module in the `./templates` folder.
+
+``` javascript
+{{example.js}}
+```
+
+# templates/
+
+* a.js
+* b.js
+* c.js
+* foo.js
+* baz.js
+* bar.js
+
+All templates are of this pattern:
+
+``` javascript
+module.exports = function() {
```

Comment:
Please reference files here with the `{{templates/b.js}}` notation. Don't repeat code.